microsoft-edge/devtools-guide-chromium/contact.md

```diff
@@ -0,0 +1,31 @@
+---  
+description: Contacting the Microsoft Edge DevTools team.
+title: Contacting the Microsoft Edge DevTools team
+author: MSEdgeTeam  
+ms.author: msedgedevrel  
+ms.date: 08/06/2021  
+ms.topic: conceptual  
+ms.prod: microsoft-edge  
+ms.technology: devtools  
+keywords: microsoft edge, web development, f12 tools, devtools  
+---  
+# Contacting the Microsoft Edge DevTools team
```

Comment:
The Scope question: why "DevTools"?  Should this instead be "the Microsoft Edge development team", corresponding to the entire TOC for Microsoft Edge Developer documentation?

Link to this article from all docs in the doc-set?
Or, conversely, remove the link to this page from all DevTools pages?
Link to this page from just a few strategic articles?

Affected lines: Globally find "DevTools" in this page, and in the hot text of all the links to this page.

Reply:
Resolved: promoted the new page and made it cover the entire TOC.